lib/Parser/Hash.h

```diff
@@ -394,7 +394,6 @@ class HashTbl
         m_perr = perr;
         memset(&m_rpid, 0, sizeof(m_rpid));
     }
-    ~HashTbl(void) {}
```

Comment:
I think this is to enforce that new/delete are not called on HashTbl, but uses Create/Release instead.
We probably can keep the destructor private that if we override new/delete of the HashTbl class and put that private as well.